mail_multicompany_force_company/README.rst

```diff
@@ -0,0 +1,93 @@
+
+
+================================
+Email Multi Company Force Server
+================================
+
+This module forces to use the mail server that is configured for the company in outgoing mail server configuration.
+If there is a mail server without a company assigned, it will be used by all companies.
+In case there is a mail server without company and your company has assigned a mail server, you will use your company's mail server
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fmulti--company-lightgray.png?logo=github
+    :target: https://github.com/OCA/multi-company/tree/15.0/mail_multicompany
+    :alt: OCA/multi-company
+
+|badge1| |badge2| |badge3|
+
+This module forces to use the mail server that is configured for the company in outgoing mail server configuration.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
```

Comment:
This should be in one of the `readme/*.rst` files, then the README.rst will be autogenerated